src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Endpoint.kt

```diff
@@ -31,7 +34,7 @@ data class Endpoint(
 
     override fun additionalSetup(url: String)
     {
-        if (requiredPermissions.any())
```

Comment:
Now that `can-login` is gone, we might have no specific permission requirements but still want an endpoint to be behind authentication, hence this new `secure` flag.